test-cases/tests/alerts/c18-validate-api-usage-alerts.md

```diff
@@ -14,148 +14,126 @@ tags:
+   }'
+   ```
+
+5. Patch the `rhoam` CR to specify BU, SRE and Customer email addresses:
+
+   NOTE: Replace `<rh_username>` references in the below commands with a valid Red Hat username. For example: `pamccart+BU@redhat.com`
+
+   Patch BU and SRE email addresses:
+
+   ```sh
+   oc patch rhmi rhoam -n redhat-rhoam-operator --type merge -p '{"spec":{"alertingEmailAddresses":{"businessUnit":"<rh_username>+BU@redhat.com", "cssre":"<rh_username>+SRE@redhat.com"}}}'
+   ```
+
+   Patch Customer email addresses with two emails to validate multiple addresses:
+
+   ```sh
+   oc patch rhmi rhoam -n redhat-rhoam-operator --type merge -p '{"spec":{"alertingEmailAddress":"<rh_username>+CUSTOMER1@redhat.com <rh_username>+CUSTOMER2@redhat.com"}}'
+   ```
+
+6. Verify email addresses are added to Alert Manager configurations
+
+   Open the Alert Manager console and login via Openshift:
+
+   ```sh
+   open "https://$(oc get route alertmanager-route -n redhat-rhoam-middleware-monitoring-operator -o jsonpath='{.spec.host}')"
+   ```
+
+   Go to `Status -> Config` and check that the BU, SRE and Customer email addresses are included in the `BUAndCustomer` and `SRECustomerBU` receivers in the Alert Manager configuration where appropriate.
+
+7. From the Openshift console, retrieve the 3Scale admin password by going to `Secrets` > `system-seed` under the `redhat-rhoam-3scale` namespace and copying the `admin` password
+
+8. Next, open the 3Scale admin console and login with `admin` as the username and the retrieved password
+
+   ```sh
+   open "https://$(oc get route -n redhat-rhoam-3scale | grep 3scale-admin | awk {'print $2'})"
+   ```
+
+9. Click on `Ok, how does 3scale work?` and follow the 3Scale wizard to create an API
+10. Once on your API overview page, click on `Integration` on the left, then on `Configuration`
+11. Take note of the `example curl for testing` for `Staging-APIcast`
+12. Trigger API Usage alerts
+
+    | Alert to fire                        | `numRequests` | `interval` | `recipients`          |
+    | ------------------------------------ | ------------- | ---------- | --------------------- |
+    | RHOAMApiUsageLevel1ThresholdExceeded | 200           | 0.48       | BU and Customers      |
+    | RHOAMApiUsageLevel2ThresholdExceeded | 200           | 0.43       | BU and Customers      |
+    | RHOAMApiUsageLevel3ThresholdExceeded | 200           | 0.4        | BU, Customers and SRE |
```

Comment:
@pmccarthy what's the motivation behind changing these intervals and the script? They worked in previous release testing, but I cannot make the alerts firing with the new script and new values.

Reply:
@psturc I was able to get the alerts to fire yesterday with the intervals that I included in the comments. I think that these intervals are going to be different in each case so we probably need to update the test-case to tell the executor that they will need to try different intervals. The way that I did it was start with 0.60 as the interval and then check in prometheus what number it was at. In my case it was aroun 60 so I kept reducing the interval and running the command until I got to 82 and then to 86 and then to 92. I think we'd need to write steps for the verify to do that rather than a specific set of intervals. cc @pmccarthy

Reply:
After discussing it with @psturc I'm also in favour of reverting to the old script which was working before. I found the above to be a very lengthy process and probably will be each time.

Reply:
I've made a number of updates and am trying a new approach for triggering the alerts that doesn't require intervals. Maybe we can see how it goes and if it's still not working we can revert back to the original script.

Personally I'm not a fan of creating a script as I'm lazy. Copying a one liner seems easier to me :)

Reply:
Hey @pmccarthy I had a cluster up so I checked if the commands trigger the alert and I was able to reliably trigger the alerts following the steps here. I'm cool with these new steps. 👍  will come back to the other verification Monday.